Robot/ConversationLibraries/SpeechTools/GoogleSpeech/GoogleSpeechService.cs

```diff
@@ -180,7 +197,7 @@ private async Task<SpeechToTextData> TranslateAudioStreamInternal(byte[] audioDa
 				}
 				else
 				{
-					ISendExternalRequestResponse sdata = await _robot.SendExternalRequestAsync("POST", _servicesAuthorization.STTEndpoint + _servicesAuthorization.SubscriptionKey, "BEARER", _servicesAuthorization.SubscriptionKey, arguments, false, false, null, "application/json");
+					ISendExternalRequestResponse sdata = await _robot.SendExternalRequestAsync("POST", _recognitionServicesAuthorization.Endpoint + _recognitionServicesAuthorization.SubscriptionKey, "BEARER", _recognitionServicesAuthorization.SubscriptionKey, arguments, false, false, null, "application/json");

```

Comment:
Weird, what made you use this?

Reply:
I think this is a failover, I made this service so long ago, it could use some cleanup.